docs/Homebrew-on-Linux.md

```diff
@@ -5,7 +5,10 @@ redirect_from:
   - /Linux
   - /Linuxbrew
 ---
-<img src="https://brew.sh/assets/img/linuxbrew.png" alt="Linuxbrew logo" style="min-width:128px" height="128" width="128">
+
+<p style="text-align:center">
```

Comment:
Have you tested this locally? This won't do a centre align of the image.

Reply:
Yes I have tested it locally, and yes it does centre the image. I guess `text-align:center` is a misnomer, since it centre aligns all inline elements in the paragraph, including images, not just text.

Reply:
My apologies, yes, this works as expected. It still feels a bit gross having the raw HTML here. What would you think about a YAML frontmatter option that replaced the image at the top of the page instead? It does feel a little weird right now to have a Homebrew logo and then an identically sized Linuxbrew logo below it. Could also be nice for e.g. the @BrewTestBot page too.

Reply:
Yep, I like the idea. I don't have the time for it right at this moment, but definitely a good wishlist item. Since this PR is hardly any worse than what we have currently, can we merge this PR as is?

Reply:
Done in https://docs.brew.sh/Homebrew-on-Linux

Reply:
Thanks, Mike!